Altis_Life.Altis/core/actions/fn_processAction.sqf

```diff
@@ -8,9 +8,12 @@
     Master handling for processing an item.
     NiiRoZz : Added multiprocess
 */
-private ["_vendor","_type","_itemInfo","_oldItem","_newItemWeight","_newItem","_oldItemWeight","_cost","_upp","_hasLicense","_itemName","_oldVal","_ui","_progress","_pgText","_cP","_materialsRequired","_materialsGiven","_noLicenseCost","_text","_filter","_totalConversions","_minimumConversions"];
-_vendor = [_this,0,objNull,[objNull]] call BIS_fnc_param;
-_type = [_this,3,"",[""]] call BIS_fnc_param;
+private ["_hasLicense","_materialsRequired","_materialsGiven","_noLicenseCost","_text","_filter"]
+params [
+    ["_vendor",objNull,[objNull]],
+    ["_type","",[""]]
```

Comment:
Not how it works, _type is index 3, you've put it as index 1,
it'd be done like
params [
    ["_vendor",objNull,[objNull]],
    "",
    "",
    ["_type","",[""]]
];